testgen/generators/common.js

```diff
@@ -0,0 +1,16 @@
+
+function xsample_tests(all_tests, run_limit) {
```

Comment:
Is the spelling of `xsample_tests` intentional?

Reply:
Fixing this. Thx.